content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -150,95 +145,95 @@ You should see a confirmation message from sensuctl.
 Created
 {{< /highlight >}}
 
-The `sensu-slack-handler` asset is now ready to use with Sensu.
-You can use sensuctl to see the complete asset definition.
+The `sensu-slack-handler` asset is now ready to use with Sensu.  Use sensuctl to see the complete asset definition.
 
 {{< highlight shell >}}
 sensuctl asset info sensu-slack-handler --format yaml
 {{< /highlight >}}
 
-_PRO TIP: You can use resources definition to create and update resources (like assets) using `sensuctl create --file filename.yaml`. See the [sensuctl docs][3] for more information._
+_PRO TIP: You can use resource definitions to create and update resources (like assets) using `sensuctl create --file filename.yaml`. See the [sensuctl docs][5] for more information._
 
 **3. Create a Sensu Slack handler**
 
 Open the `sensu-slack-handler.json` handler definition provided with the sandbox, and edit the definition to include your Slack channel, webhook URL, and the `sensu-slack-handler` asset.
```

Comment:
I think we should somehow incorporate instructions for handling this in vim. Edgar walked me through it, and I drafted the rough instructions listed below as result of his help.

I would like to include these instructions right in this article (and with this step), if no one objects. If anyone thinks it's too disruptive to the flow, then I think we could add a "footnote" (just a section at the bottom of the page) -- add a link here that lets people hop down to that footnote, then a link after the instructions so they can hop back to their place.

`vi sensu-slack-handler.json` (opens json file you need to edit in Vim)
`i` (to be able to write over text)
Use arrows to navigate. No copy/paste; delete and retype:
 - Replace KEEPALIVE_SLACK_WEBHOOK value with your webhook URL
 - Replace KEEPALIVE_SLACK_CHANNEL value with your slack channel
 - In the runtime_assets [], type `"sensu-slack-handler"` (including quotation marks)
Press esc
Type `:w` (to write/save your changes)
Type `:q` (to quit back to sensuctl)

Reply:
While I like having more instructions for users, this would assume they are using `vim` when they can be using `nano`, some other text editor or even be using Windows. I sorta lean towards a tool like `sed` for this, but that assumes still that they are on a Linux based system (and even given that there are GNU vs non-GNU differences). Maybe we could just add more English like `Open the handler definition in your favorite text editor`.

Reply:
I added the clarification as you suggested. This came up because I didn't know what to do at this point, and Edgar walked me through using vim (I think b/c it was the easiest thing to use at the time). Edgar mentioned this is something a typical sysadmin would know how to do.

What do you think of something like...adding the vim instructions as a gist that we can link to from here? I think the instructions are too detailed to insert right in the text, but I would not have been able to continue the lesson if Edgar hadn't been available to help me figure it out.

Reply:
If we really want to have instructions for just `vim`, we would need to at least call out "vim or vi" as not everyone uses `vim` (vim=vi improved). I for one do not use it on OSX cause of some odd unpredictable behavior.

However the rest of the actions should be the same. A gist sounds fine to keep the guide cleaner.

Reply:
Would it be better to give instructions for a different text editor? I really just want to get a totally new user through this step...the only reason I have vim listed is because that's how Edgar showed me (and you can just do it within the rest of the workflow)

Reply:
Also maybe Edgar actually showed me in `vi`

Reply:
I think `vi` and `vim` is fine for now. If we get feedback about people getting stuck, we can add a different editor or method.

Any other tool we pick or add is going to have the same problem; they might not have it. Having at least one example is a good idea.